db/institutions.yml

```diff
@@ -420,4 +420,9 @@ uduisburgessen:
   state: North Rhine-Westphalia
   street: Universitaetsstrasse 7
   zip: 45117
-
+bayreuth:
+  aka: [Bayreuth, bayreuth]
+  city: Bayreuth
+  country: Germany
+  departments: {chemistry: Biology, Chemistry & Earth Sciences}
+  name: Universitat Rayreuth
```

Comment:
typo?

Reply:
also, if we are going with the German naming I think we should make sure to get the accents right.  Please double-check.

Reply:
Yea there needs to be a double dot over the A. (I guess I need to look up how to put unicode into things)

Reply:
copy-paste off the website seems to work for the other guys!   It means we need utfenc package (or sthg like that, I have a note somewhere)......

Note also the "Rayreuth"

Reply:
Yes I saw the "R". I'll get them both.